Login.Component.html

```diff
@@ -0,0 +1,118 @@
+<html>
+<head>
+	<title> J Boss </title>
+<link href="indexcss.css" rel="stylesheet">
+
+
+<!-- Script -->
+<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js"></script>
+<!-- Script closed -->
+</head>
+<body>
+	<div class="log">
+	<body background="images/bg-01.jpg">
+
+
+
+<center>
+	<img src="Capture.png">
+</center>
+
+<div class="login-page">
+<div class="form">
+<form class="register-form">
+
+
+
+
+<!-- Angular JS -->
+<!-- Tags of google code in only told to put angular -->
+
+<div ng-app="">
+
+  Username <input type="username" name="user" ng-model="user"
+type="username" placeholder="username"/>
+
+  Password <input type="password" name="per" ng-model="par"
+type="password" placeholder="password"/>
+
+
+
+
+
+Email Id<input name="myInput" ng-model="myInput" type="email"
+ng-model="email" placeholder="email id" />
+
+
+
+
+<button ng-click="">Create</button>
+</div>
+<!-- Angular JS  closed-->
+</div>
+</div>
+</form>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+     
+
+   <footer class="footer">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-8 col-md-10 mx-auto">
+            <ul class="list-inline text-center">
+              <li class="list-inline-item">
+                <a href="www.twitter.com">
+                  <span class="fa-stack fa-lg">
+                    <i class="fas fa-circle fa-stack-2x"></i>
+                    <i class="fab fa-twitter fa-stack-1x fa-inverse"></i>
+                  </span>
+                </a>
+              </li>
+              <li class="list-inline-item">
+                <a href="www.facebook.com">
+                  <span class="fa-stack fa-lg">
+                    <i class="fas fa-circle fa-stack-2x"></i>
+                    <i class="fab fa-facebook-f fa-stack-1x fa-inverse"></i>
+                  </span>
+                </a>
+              </li>
+              <li class="list-inline-item">
+                <a href="www.github.com">
+                  <span class="fa-stack fa-lg">
+                    <i class="fas fa-circle fa-stack-2x"></i>
+                    <i class="fab fa-github fa-stack-1x fa-inverse"></i>
+                  </span>
+                </a>
+              </li>
+            </ul>
+            <p class="copyright text-muted">Copyright &copy; JBoss Community 2018</p>
+          </div>
+        </div>
+      </div>
+    </footer>
+<script src="bootstrap/js/jquery.js"></script>
+<script src="bootstrap/js/bootstrap.js"></script>
```

Comment:
Please don't include any script src... bootstrap is already present in the project. Also, we don't need angular.min.js ...else this is good to go